src/devices/Mcp3xxx/Mcp3xxx.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Spi;
+
+namespace Iot.Device.Adc
+{
+    /// <summary>
+    /// Mcp3xxx Abstract class representing the MCP 10 and 12 bit ADC devices.
```

Comment:
MCP family of ADC devices (I'd skip number of bits especially some of these are 13-bits from what I've seen)